### PR TITLE
Cart empty #107 and User Profile displays Orders #119

### LINKED ISSUE #107: User Story 29, User can empty a cart that has items
```
As a visitor or registered user
When I have items in my cart
And I visit my cart ("/cart")
And I click the link to empty my cart
Then I am returned to my cart
All items have been completely removed from my cart
The navigation bar shows 0 items in my cart
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #111

depends on #109

### LINKED ISSUE #119: User Story 18, User Profile displays Orders
```
As a registered user
When I visit my profile page
If my account has ordered items
I see every order I've made, which includes the following information:
- the ID of the order, which is a link to the order show page
- the date the order was made
- the date the order was last updated
- the current status of the order
- the total quantity of items in the order
- the grand total of all items for that order
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #121

depends on #120

depends on #102

### PR DESCRIPTION
**Pull request by stoic-plus**
closes #107 and closes #119 

* Adds `empty_cart` to app controller
* destroy route to cart item controller
* link to empty cart on /cart
* tests for page interaction and correct content to reflect cart status

__Whoops!__

Forgot to make another branch
* Also rendering a default user's order information (the following for each order)
  * Id (is a link to order show page)
  * Date (created and updated at)
  * Status
  * Total item quantity
  * Grand total

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -43,6 +43,8 @@
   create_table "orders", force: :cascade do |t|
     t.string "status"
     t.bigint "user_id"
+    t.datetime "created_at", null: false
```
Comment (le3ah) [inline db/schema.rb]: I thought we were leaving the timestamps off of the order table since we don't need them.  Either way is fine, but it might get confusing if we have them there.

Comment (stoic-plus) [reply db/schema.rb]: Card #119 says to include them

```
- the date the order was made
- the date the order was last updated
```

I guess we could somehow go off of order items but that what would be more complicated than this

Comment (le3ah) [reply db/schema.rb]: Oh, okay, cool.  Hooray!  Great job!
